src/coins.h

```diff
@@ -230,6 +230,40 @@ class CCoinsViewCursor
     uint256 hashBlock;
 };
 
+struct CoinsViewCacheCursor
+{
+    //! will_erase should be set to true in CCoinsViewCache::Flush and false in CCoinsViewCache::Sync
```

Comment:
this seems redundant, if we're not sure about this, I'd rather add a test that can bite back (as you can tell, I'm not a big fan of dead comments :p)

Reply:
Updated the comment to @sipa's version.